src/main/java/com/dataiku/dss/model/metadata/DssRecipeMetadata.java

```diff
@@ -1,6 +1,6 @@
 package com.dataiku.dss.model.metadata;
 
-public class DssRecipeMetadata {
+public class DssRecipeMetadata{
```

Comment:
Nitpick: I like the whitespace before `{` :)

Reply:
Didnt see this get through, I agree with you, I think modifying existing code just for small style points makes a PR more confusing for minimal gains. Fixed in the new PR